content/_index.de.md

```diff
@@ -10,12 +10,12 @@ title = "Das Postmeritokratische Manifest"
 
 Meritokratie ist ein grundlegendes Prinzip der Open-Source-Bewegung und das meritokratische Ideal wird im Bereich der Informationstechnologie (IT) durch die Art und Weise, in der Menschen angeworben, eingestellt, in Unternehmen gehalten, befördert und wertgeschätzt werden, fortwährend aufrechterhalten.
 
-Es hat sich allerdings immer wieder gezeigt, dass Meritokratie hauptsächlich denjenigen, die privilegiert sind, zugute kommt und damit Menschen, die in der IT-Branche unterrepräsentiert sind, ausschließt.
-Tatsächlich ist die Idee von anerkennenswerter Leistung niemals klar definiert; stattdessen scheint sie eine Form der Bestätigung zu sein, eine Anerkennung, dass "diese Person insofern wertvoll sei, als dass sie so sei wie ich."
+Allerdings hat sich immer wieder gezeigt, dass Meritokratie hauptsächlich denjenigen, die privilegiert sind, zugute kommt und damit Menschen, die in der IT-Branche unterrepräsentiert sind, ausschließt.
+Tatsächlich ist die Idee des Verdienst niemals klar definiert; stattdessen scheint sie eine Form der Bestätigung zu sein, eine Anerkennung, dass "diese Person insofern wertvoll sei, als dass sie so sei wie ich."
```

Comment:
I  also found the term you used here on [the German article on Meritocracy](https://de.wikipedia.org/wiki/Meritokratie) which translates `merit` to the German word `das Verdienst` (in contrast to `der Verdienst`):

> Eine Meritokratie (lat.: meritum, das Verdienst und griech.: κρατεῖν, kratein herrschen) [...]

In the first iteration of the translation I went with the synonym `anerkennenswerte Leistung`, 
as I thought that the original wording `das Verdienst` is not so commonly used in German and might be confusing to readers. Yet, I'm not sure if it's just me (having the feeling that the wording is uncommon in German) and I do believe `das Verdienst` is actually the more authentic translation for the English text here. So I think it's cool to update it :+1: 

If we go with `das Verdienst`, according to [duden.de](https://www.duden.de/rechtschreibung/Verdienst_Leistung_Werk_Erfolg),  the _Genitiv_ form which is used here  would then be

```
des Verdienstes
```
or 

```
des Verdiensts
```